plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -91,4 +92,22 @@ public IcebergConfig setMaxPartitionsPerWriter(int maxPartitionsPerWriter)
         this.maxPartitionsPerWriter = maxPartitionsPerWriter;
         return this;
     }
+
+    /**
+     * In case of some queries / tables, retrieving table statistics from Iceberg
+     * can take 20+ seconds. This config allows the user / operator the option
+     * to opt out of retrieving table statistics in those cases to speed up query planning.
+     */
+    @Config("iceberg.table-statistics-enabled")
```

Comment:
Let's make it consistent with the session property skip the `table`

Reply:
This is named consistently with hive's `hive.table-statistics-enabled`